openslides/motions/static/js/motions/motions.js

```diff
@@ -225,92 +385,483 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions'])
+                    optionsAttr: 'bs-options',
+                    options: Tag.getAll(),
+                    ngOptions: 'option[to.valueProp] as option in to.options | filter: $select.search',
+                    valueProp: 'id',
+                    labelProp: 'name',
+                    placeholder: gettext('Select or search a tag...')
+                },
+                hideExpression: '!model.more'
+            },
+            {
+                key: 'supporters_id',
+                type: 'ui-select-multiple',
+                templateOptions: {
+                    label: gettext('Supporters'),
+                    optionsAttr: 'bs-options',
+                    options: User.getAll(),
+                    ngOptions: 'option[to.valueProp] as option in to.options | filter: $select.search',
+                    valueProp: 'id',
+                    labelProp: 'full_name',
+                    placeholder: gettext('Select or search a supporter...')
+                },
+                hideExpression: '!model.more'
+            },
+            {
+                key: 'workflow_id',
+                type: 'ui-select-single',
+                templateOptions: {
+                    label: gettext('Workflow'),
+                    optionsAttr: 'bs-options',
+                    options: Workflow.getAll(),
+                    ngOptions: 'option[to.valueProp] as option in to.options | filter: $select.search',
+                    valueProp: 'id',
+                    labelProp: 'name',
+                    placeholder: gettext('Select or search a workflow...')
+                },
+                hideExpression: '!model.more',
+                // preselect default workflow
+                defaultValue: Config.get('motions_workflow').value
+            },
+        ];
+    }
+])
+
+.controller('MotionUpdateCtrl', [
+    '$scope',
+    '$state',
+    'gettext',
+    'Motion',
+    'Agenda',
+    'Category',
+    'Config',
+    'Mediafile',
+    'Tag',
+    'User',
+    'Workflow',
+    'motion',
+    function($scope, $state, gettext, Motion, Agenda, Category, Config, Mediafile, Tag, User, Workflow, motion) {
+        Agenda.bindAll({}, $scope, 'items');
+        Category.bindAll({}, $scope, 'categories');
+        Mediafile.bindAll({}, $scope, 'mediafiles');
+        Tag.bindAll({}, $scope, 'tags');
+        User.bindAll({}, $scope, 'users');
+        Workflow.bindAll({}, $scope, 'workflows');
+
+        // set initial values for form model
+        $scope.model = motion;
+        $scope.model.more = false;
+
+        // save form
+        $scope.save = function (model) {
+            Motion.save(model)
+                .then(function(success) {
+                    $state.go('motions.motion.list');
+                })
+                .catch(function(fallback) {
+                    console.log(fallback);
+                });
+        };
+
+        // define all update form fields
+        $scope.formFields = [
```

Comment:
You already wrote this in the CreateCtr.

Please find a method that is not redundant. In other case it is to hard to fix the form.